resfit/fit_resonator/fit_S_data.py

```diff
@@ -971,7 +971,7 @@ def min_fit(params,xdata,ydata,Method):
         quit()
 
 
-def fit_resonator(filename: str,Method,normalize: int,dir: str = None, data_array: np.ndarray = None, background: str = None, background_array: np.ndarray = None, plot_extra = True):
+def fit_resonator(filename: str,Method,normalize: int,dir: str = None, data_array: np.ndarray = None, background: str = None, background_array: np.ndarray = None, plot_extra = False):
```

Comment:
Do you mind starting a newline after each argument, so the line isn't super long? Basically:
```python

def fit_resonator(filename: str,
                  Method,
                  normalize: int,
                  dir: str = None, 
                  data_array: np.ndarray = None, 
                  background: str = None, 
                  background_array: np.ndarray = None, 
                  plot_extra = False):
```

Reply:
Will do!